proto/control-api/src/grpc/api.proto

```diff
@@ -4,6 +4,8 @@ syntax = "proto3";
 
 package api;
 
+import "google/protobuf/duration.proto";
```

Comment:
Now `make cargo.gen crate=medea-control-api-proto` generates empty file `proto/control-api/src/grpc/google.protobuf.rs`. 

It should be either not generated at all, or included under VCS, or properly `.gitignore`d.